_pages/algorithm-development/languages/python.md

```diff
@@ -306,41 +240,28 @@ And then raise the error with a helpful error message:
 raise AlgorithmError("Please provide a valid URL")
 {% endhighlight %}
 
-If you are creating an algorithm that relies on calling another algorithm you may use Algorithmia error messages for catching errors thrown by that algorithm:
+Additionally, if you are creating an algorithm that relies on calling another algorithm you may use Algorithmia error messages for catching errors thrown by that algorithm:
 
 {% highlight python %}
 import Algorithmia
 
-client = Algorithmia.client()
-# An algorithm that takes a URL
-algo = client.algo("util/Url2Text/0.1.4")
-
-try:
-  	print(algo.pipe(input).result)
-except Exception as error:
-    print(error)
+def apply(input):
+    try:
+        print(algo.pipe(input).result)
+    except Exception as error:
+        print(error)
+
+def load():
+    client = Algorithmia.client()
+    # An algorithm that takes a URL
+    algo = client.algo("util/Url2Text/0.1.4")
 {% endhighlight %}
 
 For more information on error handling see the <a href="{{site.baseurl}}/algorithm-development/algorithm-basics/algorithm-errors">Better Error Handling Guide</a>.
 
-## Algorithm Checklist
+## Algorithms with multiple files
 
-Before you are ready to publish your algorithm it's important to go through this [Algorithm Checklist]({{site.baseurl}}/algorithm-development/algorithm-checklist) and check out this blog post for <a href="https://algorithmia.com/blog/advanced-algorithm-design">Advanced Algorithm Development <i class="fa fa-external-link"></i></a>.
-
-Both links will go over important best practices such as how to create a good algorithm description, add links to external documentation and other important information.
-
-## Publish Algorithm
-
-Once you've developed your algorithm, you can publish it and make it available for others to use.
-
-To learn how to publish your algorithm: <a href="{{site.baseurl}}/algorithm-development/your-first-algo">creating and publishing your algorithm</a>
-
-## Caveats
-#### Algorithms with multiple files
-Putting everything in one source file sometimes doesn't make sense and makes stuff hard to maintain, many times you'll want to break your code into multiple source files.
-
-Importing your secondary files contains some ceavats when
-executing your algorithm on Algorithmia, in particular the import paths you use locally may vary from ours.
+Putting everything in one source file sometimes doesn't make sense and makes code more difficult to maintain, so in many cases you'll want to break your code into multiple source files. The Algorithmia platform supports using multiple source files, but you'll need to be aware that the import paths you use locally may differ from ours.
```

Comment:
An example would be nice here

Reply:
Just my two cents, this sort of strays into "how to write good code" vs "how to use our platform". I think an example would be nice, but maybe not essential/priority for getting this page published, and we could make a ticket to add it when we think of something.

Maybe it would suffice to insert a description of an example use case, after the first sentence, like: "For example, you might have several utility functions related to establishing a connection with an external database and then reading and writing data using that connection. To keep your code modular, you might define this functionality in a separate file that you then import into your main algorithm file where the functions are actually called."

Reply:
https://gist.github.com/zeryx/14ef2946033b60b9fccebe9112999dec - specifically the `from src.xxx` is the part that is different from typical local multi-file development in python, source files live in `src` and our execution directory is the projects root directory, so we need to reference src, or use a `.xxx` which implies a relative import